src/pages/StocktakesPage.js

```diff
@@ -79,10 +80,10 @@ export class StocktakesPage extends GenericPage {
     const { stocktakes, selection } = this.state;
     const { database } = this.props;
     database.write(() => {
-      const stocktakesToDelete = [];
+      let stocktakesToDelete = [];
       for (let i = 0; i < selection.length; i++) {
         const stocktake = stocktakes.find(s => s.id === selection[i]);
-        if (stocktake.isValid() && !stocktake.isFinalised) stocktakesToDelete.push(stocktake);
+        if (stocktake.isValid() && !stocktake.isFinalised) stocktakesToDelete = stocktake;
```

Comment:
Oh right I would have done this when I was testing realm. They changed the toString method of objects, which broke our re-implementation of database.delete. Will revert this!